docs/modular-resources/create/_index.md

```diff
@@ -274,6 +277,7 @@ import (
     "go.viam.com/rdk/components/motor"
     "go.viam.com/rdk/resource"
     "go.viam.com/rdk/spatialmath"
+    "go.viam.com/utils"
```

Comment:
```suggestion
```

Unless you are *directly using* a import in go, it will fail to compile.

We already import `golog` so its okay